package.json

```diff
@@ -269,6 +269,7 @@
     "re-resizable": "^6.2.0",
     "react": "17.0.1",
     "react-beautiful-dnd": "13.0.0",
+    "react-color": "2.18.0",
```

Comment:
Is this a mistake? I can't see a corresponding yarn.lock file change.

Reply:
this is on purpose. this was already required inside grafana-ui, but i also required it at root so that when it gets bundled it's within `vendors/node_modules` like everything else, instead of in `vendors/packages/grafana-ui/node_modules`. it was the only dep that was remaining in that bucket, so this change moves it into the same location as the rest, otherwise it's a thin squished line that has no labels until you zoom in a ton.

Reply:
Ok, thanks for clarification.